src/test/rrutil.h

```diff
@@ -106,6 +106,18 @@ typedef unsigned char uint8_t;
 #define ALEN(_a) (sizeof(_a) / sizeof(_a[0]))
 
 /**
+ * Allocate new memory of |size| in bytes. The pointer returned is never NULL.
+ * This calls aborts the program if the host runs out of memory.
+ */
+void *xmalloc(size_t size) {
```

Comment:
No, it was meant to be `inline static`. I fixed and squashed it. Thank you for catching it.